node/src/components/consensus/protocols/highway.rs

```diff
@@ -648,19 +651,29 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
             // We're ahead.
             match state.panorama().get(vid) {
                 None => {
-                    error!(?vid, "received a request for non-existing validator");
+                    warn!(?vid, "received a request for non-existing validator");
                     vec![]
                 }
                 Some(observation) => match observation {
-                    Observation::None | Observation::Faulty => {
-                        error!(
+                    Observation::None => {
+                        warn!(
                             ?vid,
                             our_next_seq,
                             ?observation,
-                            "`IndexPanorama` doesn't match `state.panorama` for validator"
+                            "expected unit for validator but found none"
                         );
                         vec![]
                     }
+                    Observation::Faulty => {
+                        let ev = match state.maybe_evidence(vid) {
+                            Some(ev) => ev.clone(),
+                            None => {
+                                error!(?vid, instance_id=?self.highway.instance_id(), "panorama marked validator as faulty but no evidence was found");
```

Comment:
```suggestion
                                error!(
                                    ?vid, instance_id = ?self.highway.instance_id(),
                                    "panorama marked validator as faulty but no evidence \
                                    was found"
                                 );
```

Reply:
This one might actualy be just a warning: What if they are marked as faulty based on evidence from the previous era?
(Ideally we'd send _that_ to them instead.)

Reply:
Do we change panorama's entries based on evidences from previous eras? I will dial the logging level down to WARN.

Reply:
Yes, we do. :+1: